resources/js/views/global/GlobalMapContainer.vue

```diff
@@ -27,13 +37,29 @@ export default {
         {
             return this.$store.state.globalmap.loading;
         }
+    },
+    methods: {
+        resizeHandler () {
+            console.log(window.innerHeight);
+            this.mapHeight = window.innerHeight - 72 + "px";
+        },
+        isPhone(){
+            if(/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)){
+                window.addEventListener("resize", this.resizeHandler);
+                return true;
+            }
+            return false;
+        }
```

Comment:
```javascript
isMobile ()
{
     return (Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
}
```

Reply:
the if statement needs to stay there to allow for the event handler when on mobile